lib/stats.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as database from '@bedrock/mongodb';
+import dayjs from 'dayjs';
+
+/**
+ * Retrieves statistics from the database within a specified date range or
+ * for a specific date up to the end of the current date.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {string} options.startDate - The start date of the range.
+ * @param {string} [options.endDate] - The end date of the range which is
+ *   optional. If an end date is not provided, the timestamp that corresponds
+ *   to the end of the day specified by Date.now() will be used as the
+ *   end date.
+ *
+ * @returns {object} An object containing statistical data.
```

Comment:
Pattern for async functions that return a promise.

https://github.com/digitalbazaar/jsonld-signatures/blob/567c8cf4479d2d8b6e65aa5e571c55d43573722b/lib/jsonld-signatures.js#L46

Reply:
Fixed pattern here [d228c44](https://github.com/digitalbazaar/bedrock-tokenization/pull/105/commits/d228c44600a77e49ea502584deef6dc15140521e)